src/components/PlaybackControls/PlaybackControls.tsx

```diff
@@ -1,4 +1,5 @@
 import { Box, Text, useFocus, useFocusManager, useInput } from 'ink';
+import FocusContext from 'ink/build/components/FocusContext';
```

Comment:
what's this?

Reply:
vscode auto import that was forgotten about